e2e/cases/assets-retry/index.test.ts

```diff
@@ -0,0 +1,114 @@
+import { expect, test } from '@playwright/test';
+import { dev, gotoPage, proxyConsole } from '@e2e/helper';
+import { pluginReact } from '@rsbuild/plugin-react';
+import { pluginAssetsRetry } from '@rsbuild/plugin-assets-retry';
+import type { PluginAssetsRetryOptions } from '@rsbuild/plugin-assets-retry';
+import type { RequestHandler } from '@rsbuild/shared';
+
+function stripAnsi(content: string) {
+  const pattern = [
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]+)*|[a-zA-Z\\d]+(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-ntqry=><~]))',
+  ].join('|');
+
+  const regex = new RegExp(pattern, 'g');
+
+  return content.replace(regex, '');
+}
+
+function count404Response(logs: string[]) {
+  let count = 0;
+  for (const log of logs) {
+    if (stripAnsi(log).includes('404')) {
```

Comment:
it seems like `stripAnsi` is not necessary here

Reply:
deleted, the current octal color code may not reach 404.